### ISSUE TITLE
License terms probably more lenient than artistic

### ISSUE THREAD

**Issue by quanah** [jonathanstowe/TermReadKey#11]
Given the original wording was:

```
            Unlimited distribution and/or modification is allowed as long as this
             copyright notice remains intact.
```

This really sounds more like public domain, except that it keeps a copyright.  MIT license really sounds the closest, as this allows use in anything, as long as the copyright is included:

https://en.wikipedia.org/wiki/MIT_License

**Comment by quanah** [comment]
I would also note it is generally illegal to change the terms of the license w/o the permission of the original copyright owner.  I.e., changing it to artistic w/o the express permission of Kenneth Albanowski is not legal.

**Comment by quanah** [comment]
And finally, the legal team at VMware who examined this license a few years back fully put it in the MIT/BSD style licensing category, rather than Artistic, which does seem most correct.

**Comment by xenu** [comment]
Of course it is legal, he didn't change the original license, he just stacked another on top of the old one.

The only term of the original license obligated him not to remove copyright notice. He complied with it by not removing it, so obviously he didn't break that license.

This kind of relicensing is not just legal, it's also pretty common. In practice, it works only with permissive licenses (like this one), many projects "wrap" BSD/MIT licenses with GPL or even proprietary licenses. That's how many commercial Unices could "steal" code from BSD.

**Comment by xenu** [comment]
PS. While this dual-licensing is legal, it doesn't make much sense. It gives user choice between obeying original license or obeying (original license && (artistic || gpl1+).

**Comment by jonathanstowe** [comment]
Yes that's about it. I had sufficient queries about the original license terms that were inherited from the original author that it seemed to expedient to offer the dual "original license" (loose BSD style) *or* Artistic. Corporations that required a more completely worded and well understood licence can choose the Artistic license, or if you are not really all that bothered or have already been using the software in a way which may be incompatible with the artistic license then are free to continue using the original one.

In a perfect world I would have flipped it completely to the Artistic licence, however at least two factors persuaded me that it should become dual licensed.  Firstly that the software had been distributed at least since 1995 with the original license and I only became the maintainer in 2001 or 2002, at that time I had no knowledge of any previous queries about the license and it wasn't until a couple of years later that I started getting queries about the license, almost exclusively from the legal departments of large companies seeking clarification (most often a more precise classification of the style of the license.)  Most individual users, developers and e.g  Linux packager seemed to have been content with the license for a decade or so, however the license issue was in danger of taking up more time than any other maintenance that the software might require.

Secondly, and this was key, I inherited the maintenance of the module after the original author completely disappeared, a number of people had tried to contact them over a period of a couple of years and the situation came to a head when the module required some changes to continue working with future releases of Perl 5 and needed a release.  At this point I considered altering the license to Artistic but being unable to contact the oriiginal author I concluded that leaving it as it was stood a better chance of reflecting the original intent of the author which can only be known from the wording of the license, however when the number of queries about the license caused me to revisit this decision I concluded that in order to reconcile the authors original but unknown intent expressed in the words of the license and the apparent need for a more explicit and easily categorised license for some users that dual licensing would be most appropriate,

So yes, it doesn't really make a lot of sense in a strictly legal sense, but I am quite sure that the wording indicating that the user can select on of the two licenses is unambiguous and reconciles the two competing concerns in my considering the licensing scheme.  That is I believe I have an ethical responsibility to attempt to preserve the original authors otherwise unknown intent *and* I should provide a license choice that the most users can work with.

I'm not a lawyer and I am not going to engage a lawyer but I think the language is clear enough and I don't think there is any reason to change (indeed changing the wording or the terms again could further exacerbate the problems some people seem to have with it.)

Thanks.
